articles/virtual-machines/workloads/sap/deployment-guide.md

```diff
@@ -982,8 +985,11 @@ To install the Azure Extension for SAP by using Azure CLI:
 
 1. Install using Azure CLI 2.0
 
-   1. Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2].
-   1. Sign in with your Azure account:
+   1. Install Azure CLI 2.0, as described in [Install Azure CLI 2.0][azure-cli-2].   
++
+NOTE: To install AEM Extension 0.2.2 or higher, the Latest version of AZ CLI must be installed.
```

Comment:
move the note to step "Install Azure CLI AEM Extension"
"Install Azure CLI AEM Extension. Make sure to use at least version 0.2.2 or higher."